Formula/a/animdl.rb

```diff
@@ -72,13 +73,13 @@ class Animdl < Formula
   end
 
   resource "idna" do
-    url "https://files.pythonhosted.org/packages/bf/3f/ea4b9117521a1e9c50344b909be7886dd00a519552724809bb1f486986c2/idna-3.6.tar.gz"
-    sha256 "9ecdbbd083b06798ae1e86adcbfe8ab1479cf864e4ee30fe4e46a003d12491ca"
+    url "https://files.pythonhosted.org/packages/21/ed/f86a79a07470cb07819390452f178b3bef1d375f2ec021ecfc709fc7cf07/idna-3.7.tar.gz"
+    sha256 "028ff3aadf0609c1fd278d8ea3089299412a7a8b9bd005dd08b9f8285bcb5cfc"
   end
 
   resource "lxml" do
-    url "https://files.pythonhosted.org/packages/2b/b4/bbccb250adbee490553b6a52712c46c20ea1ba533a643f1424b27ffc6845/lxml-5.1.0.tar.gz"
-    sha256 "3eea6ed6e6c918e468e693c41ef07f3c3acc310b70ddd9cc72d9ef84bc9564ca"
+    url "https://files.pythonhosted.org/packages/70/bb/7a2c7b4f8f434aa1ee801704bf08f1e53d7b5feba3d5313ab17003477808/lxml-4.9.1.tar.gz"
+    sha256 "fe749b052bb7233fe5d072fcb549221a8cb1a16725c47c37e42b0b9cb3ff2c3f"
```

Comment:
Not sure why this downgraded

Reply:
```
$ pipgrip --tree animdl
animdl (1.7.27)
├── anchor-kr<0.2.0,>=0.1.3 (0.1.3)
├── anitopy<2.2.0,>=2.1.0 (2.1.1)
├── click<8.2.0,>=8.0.4 (8.1.7)
├── comtypes<1.2.0,>=1.1.11 (1.1.14)
├── cssselect<1.3,>=1.1 (1.2.0)
├── httpx<0.24.0,>=0.23.0 (0.23.3)
│   ├── certifi (2024.2.2)
│   ├── httpcore<0.17.0,>=0.15.0 (0.16.3)
│   │   ├── anyio<5.0,>=3.0 (4.3.0)
│   │   │   ├── idna>=2.8 (3.7)
│   │   │   └── sniffio>=1.1 (1.3.1)
│   │   ├── certifi (2024.2.2)
│   │   ├── h11<0.15,>=0.13 (0.14.0)
│   │   └── sniffio==1.* (1.3.1)
│   ├── rfc3986[idna2008]<2,>=1.3 (1.5.0)
│   │   └── idna (3.7)
│   └── sniffio (1.3.1)
├── lxml==4.9.1 (4.9.1)
├── packaging<24,>=22 (23.2)
├── pkginfo<2.0.0,>=1.9.2 (1.10.0)
├── pycryptodomex<3.15.0,>=3.14.1 (3.14.1)
├── pyyaml<7.0,>=6.0 (6.0.1)
├── regex<2022.11.0,>=2022.10.31 (2022.10.31)
├── rich<13.3.4,>=13.3.1 (13.3.3)
│   ├── markdown-it-py<3.0.0,>=2.2.0 (2.2.0)
│   │   └── mdurl~=0.1 (0.1.2)
│   └── pygments<3.0.0,>=2.13.0 (2.17.2)
├── tqdm<4.66.0,>=4.62.3 (4.65.2)
└── yarl<1.9.0,>=1.8.1 (1.8.2)
    ├── idna>=2.0 (3.7)
    └── multidict>=4.0 (6.0.5)
```

Reply:
Looks like we previously just ignored that and bumped versions ourselves: https://github.com/Homebrew/homebrew-core/pull/163439

Reply:
yeah, both lxml and yarl needs to be updated for py3.12 build compatibility

justfoolingaround/animdl#308